x-pack/test/apm_api_integration/tests/asset_services/asset_services.spec.ts

```diff
@@ -215,6 +229,42 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         expect(logsOnly?.asset.signalTypes).to.eql({ 'asset.logs': true });
         expect(logsOnly?.service.environment).not.to.be('testing');
       });
+
+      it('return traces and logs metrics for services when multi-signals', () => {
+        const multiSignalService = response.body.services.find(
+          (item) => item.service.name === 'multisignal-service'
+        );
+
+        expect(multiSignalService?.metrics.latency).to.be(1000 * 1000); // microseconds
+        expect(multiSignalService?.metrics.throughput).to.be(2);
+        expect(multiSignalService?.metrics.transactionErrorRate).to.be(0.5);
+        expect(multiSignalService?.metrics.logRatePerMinute).to.be(1);
+        expect(multiSignalService?.metrics.logErrorRate).to.be(1);
+      });
+
+      it('return traces only metrics', () => {
+        const apmService = response.body.services.find(
+          (item) => item.service.name === 'apm-only-service'
+        );
+
+        expect(apmService?.metrics.latency).to.be(1000 * 1000); // microseconds
+        expect(apmService?.metrics.throughput).to.be(2);
+        expect(apmService?.metrics.transactionErrorRate).to.be(0.5);
+        expect(apmService?.metrics.logRatePerMinute).to.be(undefined);
```

Comment:
Wouldn't be better instead of undefined to be null? 

I'm afraid that the UI will have to handle the undefined fields based on the signals. I would rather have a defined schema that the UI always expects regardless of the signals. wdyt? For example

```
"services": [
      {
        "asset": {
          "signalTypes": {
            "asset.traces": true,
            "asset.logs": false
          },
          "identifyingMetadata": [
            "service.name"
          ]
        },
        "service": {
          "name": "synth-node-0",
          "environment": "Synthtrace: traces_logs_assets"
        },
        "metrics": {
          "latency": 1000000,
          "throughput": 0.001388888904963992,
          "transactionErrorRate": 0.5,
          "logRatePerMinute": null
          "logErrorRate": null
        }
      },
```

Reply:
My plan for the `undefined` was to show the `N/A` badge on the table, meaning there's no data at all. In case we return `null` would mean that for the given time range we don't have data, so we won't need to show the badge. Does this make sense?